app/controllers/concerns/sign_in/authentication.rb

```diff
@@ -18,6 +18,7 @@ def authenticate
       @access_token = authenticate_access_token
       @current_user = load_user_object
       validate_request_ip
+      @current_user
```

Comment:
can we make these `@current_user.present`? I just want to be clear exactly what the purpose of returning here is (for a true/false of whether the auth or load_user was successful)